homeassistant/components/growatt_server/sensor.py

```diff
@@ -427,34 +541,47 @@ def __init__(self, api, username, password, device_id, growatt_type):
         self.username = username
         self.password = password
 
-    @Throttle(SCAN_INTERVAL)
+    @Throttle(MIN_TIME_BETWEEN_UPDATES)
```

Comment:
I think we should keep the standard of SCAN_INTERVAL since this is a standard in Home Assistant and allows to be overwritten by config if desired.

https://developers.home-assistant.io/docs/integration_fetching_data/

If the 5 minute scan interval is causing issues and it needs to be lowered you can lower the SCAN_INTERVAL constant

Reply:
During testing my sensor was only updating every 10min. After adding a different throttle it updated every 5min. I tested it again and it updates every 5 minutes without the adjustment. I removed it.